app/views/examples/template-wide/index.njk

```diff
@@ -0,0 +1,32 @@
+{# To change the app page width use $govuk-page-width. To create additional custom page widths use the `govuk-width-container` mixin to create a class to do this, and set the class on the template, header and footer containers as shown below. #}
```

Comment:
What're your thoughts on having a Sass test rather than an app page for this?

Reply:
If you were keen on this idea we could still add a custom container class to the existing 'custom template' example, if you wanted somewhere to visually test this class.

Reply:
@nickcolley I've added a test for the output of `@mixin govuk-width-container`.

Reply:
I've also moved the custom container classes to the custom template example as per your suggestion.